airflow/contrib/operators/gcs_download_operator.py

```diff
@@ -33,6 +35,9 @@ def __init__(
         :param filename: The file path on the local file system (where the
             operator is being executed) that the file should be downloaded to.
         :type filename: string
+        :param store_to_xcom: If this is true, the filename will be assigned to an xcom object,
```

Comment:
Rather than True/False, it should probably be: `store_to_xcom_name`. That way users can specify the name of the XCom variable that they want to use. This would allow multiple downloads in the same DAG.

If the variable is False, then XCom is skipped.